src/span_context.js

```diff
@@ -11,59 +11,6 @@
  * <trace_id, span_id, sampled> tuple).
  */
 export default class SpanContext {
```

Comment:
I can see an argument for specifically pointing to inject() and extract() in this class comment... since SpanContext no longer has any (!) methods in `ot-javascript`, it's now just a "concept" and a little hard to comprehend without the context of inject and extract.